tests/show_techsupport/tech_support_cmds.py

```diff
@@ -117,7 +117,7 @@
     "docker stats --no-stream",
     "docker ps -a",
     "docker top pmon",
-    "docker exec -it lldp{} lldpcli show statistics",
+    "docker exec lldp{} lldpcli show statistics",
```

Comment:
Can you update the description with the associated PR number? Also will this work for 201911?

Reply:
Updated the comments. 201911 will fail this test unless using 201911 sonic-mgmt branch to test.

Reply:
Should we make this backward compatible?

Reply:
sure. added.